### DISCUSSION TITLE
link to access file/folder

### DISCUSSION THREAD

**Discussion by GeorgeGeorgeGe** [rejetto/hfs#272] [Q&A]
Hi,

Is it possible to include in your next version the availability to send the user a link to access file/folder ( with password included and encrypted in it) instead of creating an account for him in the server, the same way we do in onedrive or sharepoint?

Thanks.

**Comment by rejetto** [comment]
something like that is already in the to-do list.
It's a complex feature and won't fit "next" version.

**Comment by rejetto** [comment]
check #946
